gobblin-yarn/src/test/java/org/apache/gobblin/yarn/YarnServiceTest.java

```diff
@@ -284,6 +292,16 @@ protected ContainerLaunchContext newContainerLaunchContext(Container container,
               Arrays.asList("sleep", "60000"), Collections.emptyMap(), null, Collections.emptyMap());
     }
 
+    /**
+     * Get the list of matching container requests for the specified resource memory and cores.
+     */
+    public List<? extends Collection<AMRMClient.ContainerRequest>> getMatchingRequestsList(int memory, int cores) {
```

Comment:
is this method used somewhere?

Reply:
At lines 204 and 209 above to check whether the container requests are removed.